File: back/src/main/java/org/bliz/trivial/controller/model/Game.java
package org.bliz.trivial.controller.model;

import java.io.Serializable;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Game implements Serializable {

	private static final long serialVersionUID = -941472632367634263L;

	private Integer gameCode;

	private Integer currentUser;

	private User user1;

	private User user2;

	private User user3;

	private User user4;

//	private List<QuestionsPerGame> questionsPerGames;
}

File: front/src/app/models/game.ts
import { User } from "./user";
import { QuestionsPerGame } from "./questionsPerGame";

export class Game {

    gameCode: number;
    current_player: number;
    winnerUser: User;
    loserUser: User;
    playerOne: User;
    playerTwo: User;
    questionsPerGames: QuestionsPerGame[];

    constructor(gameCode: number, current_player: number, winnerUser: User, loserUser: User, playerOne: User, playerTwo: User, questionsPerGames: QuestionsPerGame[]) {
        this.gameCode = gameCode;
        this.current_player = current_player;
        this.winnerUser = winnerUser;
        this.loserUser = loserUser;
        this.playerOne = playerOne;
        this.playerTwo = playerTwo;
        this.questionsPerGames = questionsPerGames;
    }
}